client/components/mma/cancel/cancellationSaves/ContinueMembershipConfirmation.tsx

```diff
@@ -37,19 +39,30 @@ export const ContinueMembershipConfirmation = () => {
 				`}
 			/>
 			<Stack space={4}>
-				<Heading>Thank you for continuing your membership</Heading>
-				<p>
-					No action is needed from your side. Your membership will
-					continue as normal.
+				<h2 css={headingCss}>Thank you for keeping your Membership</h2>
+				<p
+					css={css`
+						${textSans.medium()}
+					`}
+				>
+					The new price of your Membership is £7/month. <br />
+					Your first billing date will be{' '}
+					{cancellationFormatDate(`${mainPlan.chargedThrough}`)}.
```

Comment:
```suggestion
					{cancellationFormatDate(mainPlan.chargedThrough)}.
```

Reply:
I tried that previously and it didn't work ![image](https://github.com/guardian/manage-frontend/assets/115992455/98952158-cf0f-44e0-abf4-8a5c88bc3b0d)

Reply:
ah, okay. Maybe `mainPlan.chargedThrough ?? ''` would be more appropriate? Otherwise you'll effectively be rendering this
![image](https://github.com/guardian/manage-frontend/assets/114918544/7396b5a3-eb96-436e-a3de-bace7c27a87c)